Signal.xcodeproj/project.pbxproj

```diff
@@ -2914,7 +2907,6 @@
 			files = (
 				45F59A0A2029140500E8D2B0 /* OWSVideoPlayer.swift in Sources */,
 				344F249B200FD03300CFB4F4 /* SharingThreadPickerViewController.m in Sources */,
-				34F77040203F55D000C7CBB7 /* OWS107LegacySounds.m in Sources */,
```

Comment:
I suspect you're going to want to keep this migration if you want to delay upgrading legacy users to "note" until they've seen the splash, as you have it below.

Reply:
Did you see that setting the default sound now happens when the splash screen appears?

Reply:
I see this:
```
        OWSSounds.setGlobalNotificationSound(.note)
```
But this won't have any effect if you get rid of `OWS107LegacySounds` since the default will have already changed.  I guess it might affect the fallback PN sound.  We can discuss if this isn't clear...

Reply:
You're right, thanks for patiently re-explaining =)